apps/wallet-mobile/src/Dashboard/Dashboard.tsx

```diff
@@ -30,19 +36,40 @@ import {WithdrawStakingRewards} from './WithdrawStakingRewards'
 export const Dashboard = () => {
   const intl = useIntl()
   const navigateTo = useNavigateTo()
+  const governanceStrings = useGovernanceStrings()
 
   const wallet = useSelectedWallet()
   const {isLoading: isSyncing, sync} = useSync(wallet)
   const isOnline = useIsOnline(wallet)
+  const {openModal} = useModal()
 
   const balances = useBalances(wallet)
   const primaryAmount = Amounts.getAmount(balances, '')
   const {stakingInfo, refetch: refetchStakingInfo, error, isLoading} = useStakingInfo(wallet)
+  const isGovernanceFeatureEnabled = useIsGovernanceFeatureEnabled(wallet)
 
   const [showWithdrawalDialog, setShowWithdrawalDialog] = React.useState(false)
 
   const {resetToTxHistory} = useWalletNavigation()
 
+  const isParticipatingInGovernance = useIsParticipatingInGovernance(wallet)
+  const walletNavigateTo = useWalletNavigation()
+
+  const onParticipatePress = () => {
```

Comment:
convention `onXX` on props and `handleOnXX` when even handler.
```suggestion
  const handleOnParticipatePress = () => {
```